cmake/onnxruntime_common.cmake

```diff
@@ -157,6 +159,9 @@ if(APPLE)
   target_link_libraries(onnxruntime_common PRIVATE "-framework Foundation")
 endif()
 
+if(MSVC)
+  target_link_libraries(onnxruntime_common PRIVATE dxcore.lib)
```

Comment:
Does this result to more binary size and runtime dependency?

Reply:
I would have expected dxcore to be a system library and this is just a shim. dxcore.lib is ~2K.

Reply:
onnxruntime_common is a static library which does not have a link step. So how would this line of code help?